<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="apples-MacBook-Air.local" ignored="2" name="Default test(failed)" tests="3" failures="2" timestamp="2023-01-05T09:02:13 IST" time="25.787" errors="0">
  <testcase name="Test5" time="0.123" classname="TestCases.FailedTest">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestCases.FailedTest.Test5(FailedTest.java:60)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Test5 -->
  <testcase name="Test2" time="0.27" classname="TestCases.FailedTest"/>
  <testcase name="Test3" time="0.106" classname="TestCases.FailedTest">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestCases.FailedTest.Test3(FailedTest.java:46)
at java.util.ArrayList.forEach(ArrayList.java:1257)
... Removed 32 stack frames]]>
    </failure>
  </testcase> <!-- Test3 -->
  <testcase name="Test1" time="0.0" classname="TestCases.FailedTest">
    <ignored/>
  </testcase> <!-- Test1 -->
  <testcase name="Test4" time="0.0" classname="TestCases.FailedTest">
    <ignored/>
  </testcase> <!-- Test4 -->
</testsuite> <!-- Default test(failed) -->
